shards/modules/core/wires.cpp

```diff
@@ -1399,9 +1477,9 @@ struct ParallelBase : public CapturingSpawners {
   DECL_ENUM_INFO(WaitUntil, WaitUntil, 'tryM');
 
   static inline Parameters _params{
-      {"Wire", SHCCSTR("The wire to spawn and try to run many times concurrently."), IntoWire::RunnableTypes},
-      {"Policy", SHCCSTR("The execution policy in terms of wires success."), {WaitUntilEnumInfo::Type}},
-      {"Threads", SHCCSTR("The number of cpu threads to use."), {CoreInfo::IntType}}};
+      {"Wire", SHCCSTR("The Wire to copy and schedule."), IntoWire::RunnableTypes},
```

Comment:
I noticed above too, not sure we need to capitalize those but if helps clarity why not